### DISCUSSION TITLE
My GitHub account was suspended without reason. I sent a ticket to GitHub support, but haven't heard back for two months.

### DISCUSSION THREAD

**Discussion by swlee1999** [community/community#120186] [Discussions] [answered]
### Select Topic Area

Question

### Body

The ls-rain account, which is not the account I am inquiring about now, but the account I used before, has been suspended without reason. Please restore the ls-rain account.
This is the ticket number requested for restoration with ls-rain.
Ticket ID: 2659114
email : swlee19990115@gmail.com
Account Name: ls-rain 
https://github.com/ls-rain

**Comment by LiteBrite82** [comment, marked as answer]
Hi @swlee1999 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
